Ruth/Chapter_1/exercise_1.12.4

```diff
@@ -0,0 +1,8 @@
+Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
+[GCC 5.4.0 20160609] on linux2
+Type "copyright", "credits" or "license()" for more information.
+>>> print("hello")
```

Comment:
Yes, this should not since the implementation of the `print()` returns just the characters `hello`.